Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/openSAML/CertificateManagerImplTest.java

```diff
@@ -52,12 +52,12 @@
     public void setUp() {
         final HashMap<String, String> keyStoreMap = new HashMap<>();
         keyStoreMap.put(CertificateManagerImpl.KEY_STORE_KEY, KEY_STORE_PATH);
-        keyStoreMap.put(CertificateManagerImpl.KEY_STORE_PASSWORD_KEY, "changeit");
+        keyStoreMap.put(CertificateManagerImpl.KEY_STORE_PSWRD_KEY, "changeit");
         keyStoreMap.put(CertificateManagerImpl.KEY_STORE_TYPE_KEY, "JKS");
 
         final HashMap<String, String> trustStoreMap = new HashMap<>();
         trustStoreMap.put(CertificateManagerImpl.TRUST_STORE_KEY, TRUST_STORE_PATH);
-        trustStoreMap.put(CertificateManagerImpl.TRUST_STORE_PASSWORD_KEY, "changeit");
+        trustStoreMap.put(CertificateManagerImpl.TRUST_STORE_PSWRD_KEY, "changeit");
```

Comment:
Please change these back to password, it's still the most descriptive name for the constant.